app/controllers/stories_controller.rb

```diff
@@ -48,6 +48,10 @@ def show
 
   private
 
+  def assign_hero_banner
+    @hero_display_ad = DisplayAd.where(placement_area: "home_hero").first
```

Comment:
We probably want to go through the `for_display`/`filtered_ads_query` for the home hero banner as well so that all our billboards are routed through one place. In addition, if we define multiple `home_hero` ads then I'm assuming the expected functionality would probably be that we want to surface them accordingly them based on success rate.

Maybe something like this? 

```suggestion
    @hero_display_ad =   DisplayAd.for_display(area: "home_hero")
```